models/jachin_bmk.py

```diff
@@ -8,6 +8,12 @@
 
 
 class BmkTable(Base):
+    """ This class creates the table 'bmk_table' in the database using the __tablename__ method.
+        The attributes inside the Column specifies the name of the column the table will have, the
+        type of values it will have and whether the value will be a primary or foreign key. Then the
+        Column with its attributes are assigned to a variable which will be a class attribute
+        in the __init__ method. """
+
```

Comment:
lets have the docs open and close tripple quotes on leftmost and the content alighened on the same class margin
something like this

```
class BmkTable(Base):
        """ 
        This class creates the table 'bmk_table' in the database using the __tablename__ method.
        The attributes inside the Column specifies the name of the column the table will have, the
        type of values it will have and whether the value will be a primary or foreign key. 
       Then the Column with its attributes are assigned to a variable which will be a class attribute
        in the __init__ method. 
        """
```